src/nuget.config

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <packageRestore>
```

Comment:
This is definitely not needed here, and I don't think we really need it anywhere, as mentioned on the RepoStandards PR.

Reply:
I'm fine with removing it